lib/ex_doc/formatter/html/templates/sidebar_template.eex

```diff
@@ -24,7 +24,7 @@
 
   <div class="sidebar-search">
     <i class="icon-search"></i>
-    <input type="text" class="sidebar-searchInput" placeholder="search" autocomplete="off" results="0" />
+    <input type="text" class="sidebar-searchInput" placeholder="search" autocomplete="off" />
```

Comment:
@Dignifiedquire I have removed "results" was it was not a valid attribute, I think it's webkit specific. Could you please add this via Jquery? So we don't need to worry about it when we run the validators? let me know if that's still ok.